modules/ROOT/pages/flex-gateway-register.adoc

```diff
@@ -0,0 +1,26 @@
+= Register a Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Registration is used for licensing, metering and billing. You must register every flex gateway regardless of how you want to manage it.
+
+You can register your gateways in any of the following ways:
+
+* Passing a token.
+* Passing an Anypoint username and password.
+* Passing a client ID and client secret.
```

Comment:
I believe it's client ID and client secret of a Connected App – check with engineering team on the specifics.

Reply:
You can reference the instructions here: https://beta.docs.stgx.mulesoft.com/beta-flex-gateway-ga/gateway/flex-install#register-with-connected-app
It is actually "Register with a connected app" so I would remove the client Id and secret reference.

Reply:
changed to "Register with a connected app"